packages/plugin/src/rules/naming-convention.ts

```diff
@@ -105,6 +107,98 @@ const schema = {
   },
 } as const;
 
+const createRequiredPrefixNameSuggestions = (
```

Comment:
I am surprised with changed lines for such as small option. Plz do not overcomplicate this rule, use convertCase function from utils.ts

Reply:
> Plz do not overcomplicate this rule, use convertCase function from utils.ts

I see one spot in this function where `convertCase` can be used, but it'll only reduce the number of characters added in this PR by a handful. For the bulk of the logic `convertCase` is insufficient for what I think needs to be done here.

As a user, I'd expect that given configuration like this:

```ts
  'FieldDefinition[gqlType.name.value=Boolean]': {
    style: 'camelCase',
    requiredPrefixes: ['is', 'has'],
  },
```

and schema like this:

```gql
type Thing {
  enabled: Boolean
}
```

that I would be given a suggestion like `Rename to "isEnabled"`. I would be disappointed if it showed `Rename to "isenabled"`.

```ts
convertCase(`${requiredPrefix}${name}`); // -> "isenabled"
```

Unless `convertCase` is adjusted to somehow parse English into words and use that information to convert casing, I don't think that can be used to remove most lines here.

I can take a pass to see if the number of lines introduced in this PR can be reduced though.

Reply:
Ah, ok, there did end up being a way to lean on `convertCase`. Pushed!